gpoa/frontend/appliers/control.py

```diff
@@ -48,7 +48,13 @@ def get_control_status(self):
             return line.rstrip('\n\r')
 
     def set_control_status(self):
-        status = self._map_control_status(self.control_value)
+        if type(self.control_value) == int:
+            status = self._map_control_status(self.control_value)
+        elif type(self.control_value) == str:
+            status = self.control_value
```

Comment:
When mapping integer values, verification is performed, but for string values is not performed. Although we have all the necessary information.